src/components/translations/top-bar-fr.js

```diff
@@ -3,6 +3,11 @@ const top_bar_fr = {
     'top-bar/logout': 'Se déconnecter',
     'top-bar/goFullScreen': 'Plein écran',
     'top-bar/exitFullScreen': 'Quitter mode plein écran',
+    'top-bar/about': 'A propos',
+    'top-bar/displayMode': "Mode d'affichage",
```

Comment:
signle quotes

Reply:
or maybe it's ok to use double quotes for messages that have single quotes in them... maybe leave it like this then

Reply:
It's recommended by esLint. We'll have a warning if we use simple quotes here

Reply:
ok good